packages/edit-site/src/components/sidebar/spacing-panel.js

```diff
@@ -6,13 +6,20 @@ import {
 	__experimentalBoxControl as BoxControl,
```

Comment:
For another time in the future, it looks like the global styles UI could share a lot of code with the hooks (not specific to this PR). cc @nosolosw

Reply:
On the topic of sharing, there could be some benefit for the block support hooks to be able to access the default theme.json style values e.g. to reset controls correctly. From memory, I don't think these values are currently available within the post editor's state like they are in the site editor. 

So if a RangeControl is reset it might just be to `0` however the managed styles created from the theme.json might be `50`. The result is visually the control doesn't really match what is seen in the editor.

Reply:
> For another time in the future, it looks like the global styles UI could share a lot of code with the hooks (not specific to this PR). cc @nosolosw

I'd like to see that very much. I think the current struggle is that the hooks depend on the block's attributes as input ([example](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/hooks/color.js#L215)).